website/src/lib/v2-get-static-props.ts

```diff
@@ -1,9 +1,12 @@
 import { GetStaticPaths, GetStaticProps } from 'next'
+// @ts-expect-error
 import { buildDynamicMDX, buildDynamicMeta } from 'nextra/remote'
+// @ts-expect-error
 import { defaultRemarkPlugins } from '@theguild/components/next.config'
 import json from '../../remote-files/v2.json' assert { type: 'json' }
+// @ts-expect-error
 import { remarkLinkRewrite } from '@theguild/components/compile';
```

Comment:
@dotansimha should we fix type errors in components first?

Reply:
yeah we can